packages/shared/src/components/search/SearchSourceList.tsx

```diff
@@ -28,14 +28,24 @@ export const SearchSourceList = ({
   });
 
   return (
-    <PageWidgets tablet={false} className="relative order-2 laptop:order-last">
+    <PageWidgets
+      tablet={false}
+      className="relative order-3 laptop:order-last col-span-2 laptop:col-span-1"
+    >
       <i className="hidden laptop:block absolute top-8 -left-8 w-12 h-px bg-theme-divider-tertiary" />
       <div className={classNames('flex flex-col', widgetClasses)}>
         <div
           className={classNames(
-            'flex justify-between items-center py-1.5 laptop:py-4 px-4 laptop:!pb-0 laptop:bg-transparent rounded-t-16 bg-theme-bg-secondary',
+            'cursor-pointer laptop:cursor-auto flex justify-between items-center py-1.5 laptop:py-4 px-4 laptop:!pb-0 laptop:bg-transparent rounded-t-16 bg-theme-bg-secondary',
```

Comment:
Should we maybe turn this into a `button` element instead, so we won't have to manually add some props and classes.

Reply:
I thought about that, but since on desktop it's not a button I felt super conflicted 😐
Would you say just render a different wrapper on mobile rather?

Reply:
Ah, right. This is only for mobile. Then I think it would be better to keep it this way.

Reply:
Yep. Accordion component would be the best here and having a disabled state to make the header unclickable, probably.

Reply:
Fine by me, as long as the proper role and attributes are set, and from what I see they are 👍